src/cljs/commiteth/core.cljs

```diff
@@ -75,18 +75,22 @@
 (defn tabs [mobile?]
   (let [user (rf/subscribe [:user])
         owner-bounties (rf/subscribe [:owner-bounties])
-        current-page (rf/subscribe [:page])]
+        route (rf/subscribe [:route])]
     (fn tabs-render []
-      (let [tabs [[:bounties (str (when-not @user "Open ") "Bounties")]
+      (let [route-id (:route-id @route)
+            tabs [[:bounties (str (when-not @user "Open ") "Bounties")]
                   [:activity "Activity"]
                   (when (seq @owner-bounties)
```

Comment:
Is this check still desirable? My feeling was no from looking at Euge's mockups.

Within the actual dashboard page sections we could choose to display or not display data. (but the tab itself is always visible regardless of the quantity of owner-bounties)

Reply:
referring here specifically to `(when (seq @owner-bounties)`

Reply:
Good catch!

Reply:
Actually I'm not so sure about this. The dashboard isn't useful when you haven't created a bounty yet. @EugeOrtiz what do you think?